<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CHelper</class>
 <widget class="QWidget" name="CHelper">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>274</width>
    <height>153</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Mid RA:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="dmsBox" name="midRA"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Mid DE:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="dmsBox" name="midDE"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>PA:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="dmsBox" name="PA"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Width:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="dmsBox" name="scaleW"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Height:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="dmsBox" name="scaleH"/>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>72</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>dmsBox</class>
   <extends>QLineEdit</extends>
   <header>widgets/dmsbox.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
